docs/weaviate/concepts/data-import.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Data import
+sidebar_position: 11
+description: "Theoretical explanation of client-side and server-side batch imports."
+image: og/docs/concepts.jpg
+---
+
+Weaviate offers two flexible methods for importing data in bulk: **client-side batching** and **server-side batching**. This allows you to choose the best strategy based on your specific needs.
+
+- **Client-side batching** <br/>
+  In the traditional client-side approach, **your application is responsible for grouping data into batches**. You define the exact size of each batch (e.g., 100 objects) using the appropriate [client library method](../manage-objects/import.mdx). The client then sends these pre-defined chunks to the Weaviate server.
+
+  This method gives you direct control over the import process but may require manual tuning of parameters like `batch_size` and `concurrent_requests` to achieve optimal performance and avoid overwhelming the server.
+
+- **Server-side batching** <br/>
+  Server-side batching, or **automatic mode**, is a more robust and the recommended approach. Here, the client sends data as a continuous stream, and the **Weaviate server manages the data flow**.
```

Comment:
Wdyt about this? Avoiding the word `stream` as that might be misconstrued. I spoke to Tommy yesterday, and technically the data is not sent as a gRPC stream but as sets of UnaryUnary, which is the same as how insert_many does it for example.

Server-side batching, or **automatic mode**, is a more robust and the recommended approach. Here, the client sends data at a rate **based on feedback from the Weaviate server**.